src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -143,7 +150,10 @@ export const PriceOptions: React.FC<PriceOptionsProps> = ({
                   id="OfferForm_offerValue"
                   showError={showError}
                   onChange={setCustomValue}
-                  onFocus={onFocus}
+                  onFocus={() => {
+                    onFocus
```

Comment:
I think this is now missing an invocation (`onFocus()`)

Reply:
Right, thanks for spotting. Fixed it